submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md

```diff
@@ -0,0 +1,24 @@
+# Lack of Public-Key-Pins Header
+
+## Overview of the Vulnerability
+
+A lack of the HTTP response header for `Public-Key-Pins` can lead to sensitive user data being retrieved by an advanced attacker through Machine-in-the-Middle (MitM) attacks. There are multiple HTTP response headers used in communication between the server and client which can be implemented to improve security against well documented vulnerabilities. For example, the `Public-Key-Pins` security header is used in legacy versions of browsers to prevent clients from loading reflected XSS attacks.
```

Comment:
Change to Machine-in-the-Middle (MitM) to Person-In-The-Middle (PitM) would be worth here